src/utils.py

```diff
@@ -876,50 +861,37 @@ def calc_goal_h(self) -> float:
         """
         return sum([self.facts[fact].distance for fact in self.goals])
 
-    def finished(self, achieved_goals: Set[Tuple[str, ...]],
-                 queue: List[Tuple[float, float, RelaxedFact]]) -> bool:
-        """This function gives a stopping criterion for the Dijkstra search.
-        """
-        return achieved_goals == self.goals or not queue
-
-    def dijkstra(self, queue: List[Tuple[float, float, RelaxedFact]]) -> None:
-        """This function is an implementation of a Dijkstra search.
-        For efficiency reasons, it is used instead of an explicit graph
-        representation of the problem.
+    def run_forward_pass(self, initial_state: PyperplanFacts) -> None:
```

Comment:
Is the basic idea that we are doing a fixed point iteration instead of a dijkstra now, like we do for `is_dr_reachable`?Can clarify this in docstring.